modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -47,10 +47,12 @@
 	--n-tabs-title-border-radius: initial;
 	--n-tabs-title-transition: 0.3s;
 
+  	--n-tabs-icon-align-items: center;
 	--n-tabs-icon-color: var( --e-global-color-secondary, var( --n-tabs-color-secondary-fallback ) );
 	--n-tabs-icon-color-hover: var( --n-tabs-title-color-hover );
 	--n-tabs-icon-color-active: #{$white};
 	--n-tabs-icon-size: initial;
+  	--n-tabs-svg-icon-size: 15px;
```

Comment:
Why has this been added?

Reply:
To supply the SVG with an initial fixed size.

Reply:
Typography control fixed.

we can use 1 rem for the icons.
I didn't use the rems course I saw that you used px in all places.

Reply:
I have implemented the changes from your PR.
This line was deleted.